dailyjs/basic-call/components/App/Modals.js

```diff
@@ -1,10 +1,18 @@
 import React from 'react';
 import DeviceSelectModal from '@dailyjs/shared/components/DeviceSelectModal/DeviceSelectModal';
+import { useUIState } from '@dailyjs/shared/contexts/UIStateProvider';
 
-export const Modals = () => (
-  <>
-    <DeviceSelectModal />
-  </>
-);
+export const Modals = () => {
```

Comment:
Continuation of this theme seems good -- I've copied over the breakout-rooms demo based on basic-api several times, but after you started abstracting out the Aside, etc., it's made it way easier to do!